README.md

```diff
@@ -1,119 +1,23 @@
-Idris 2
-=======
+# Idris 2
 
 [![Documentation Status](https://readthedocs.org/projects/idris2/badge/?version=latest)](https://idris2.readthedocs.io/en/latest/?badge=latest)
 [![Build Status](https://github.com/idris-lang/Idris2/actions/workflows/ci-idris2.yml/badge.svg)](https://github.com/idris-lang/Idris2/actions/workflows/ci-idris2.yml)
 
-[Idris 2](https://idris-lang.org/) is a purely functional programming language
-with first class types.
+[Idris 2](https://idris-lang.org/) is a purely functional programming language with first class types.
 
-For full installation instructions, see [INSTALL.md](INSTALL.md).  Briefly, if
-you have Chez Scheme installed, with the executable name `chez`, type:
+For full installation instructions, see [INSTALL.md](INSTALL.md).
 
-* `make bootstrap SCHEME=chez`
-* `make install`
+The [wiki](https://github.com/idris-lang/Idris2/wiki) lists a number of useful resources, in particular
 
-You may need to change `chez` to be the local name of your Chez Scheme. This
-is often one of `scheme`, `chezscheme` or `chezscheme9.5` (depending on the
-version). On a modern desktop machine, this process (including tests)
-should take less than 5 minutes.
++ [What's changed since Idris 1]()
++ [Resources for learning Idris](https://github.com/idris-lang/Idris2/wiki/2-%5BCommunity%5D-Resources#learning)
++ [Editor support](https://github.com/idris-lang/Idris2/wiki/1-%5BLanguage%5D-Editor-support)
 
-Idris 2 is mostly backward compatible with Idris 1, with some minor exceptions.
-The most notable user visible differences, which might cause Idris 1 programs
-to fail to type check, are:
```

Comment:
I propose moving everything around "what changed since Idris 1" to either readthedocs or the wiki. I think it's tmi on the intro page

Reply:
The link to what's changed since Idris1 is here: https://idris2.readthedocs.io/en/latest/updates/updates.html
: )